### PR TITLE
Creating error app to show compiler errors in browser

### PR DESCRIPTION
**Pull request by anubhav94**
Created an error app inside the tool code that loads once the main app crashes, showing the logs that are printed to the Terminal prompt. This app persists a DDP connection to the user and allows developers to never have to go back to the terminal as the file watching api will reload their main app once they save and show the main app if the error is fixed or show the error on the error app.

Need to do:
- [x] Remove port 8600 hard coding
- [x] Remove 'localhost' hard coding for error app host
- [ ] Add in reload server button
- [ ] Hide behind production flag so loaded only in debug only
- [x] Add in app reloading update to Errors app when the error is updating

@Slava can you give this a look?

### CODE REVIEW

File: tools/commands.js [hunk 1]
```diff
@@ -392,6 +392,18 @@ function doRunCommand (options) {
     mobileServer = mobileServer + ":" + parsedMobileServer.port;
   }
 
+  let errorAppPath = files.pathJoin(files.convertToStandardPath(__dirname),
```
Comment (stubailo) [inline tools/commands.js]: Is there any reason this is `let` and not `const`? https://github.com/meteor/javascript#13.1

Comment (anubhav94) [inline tools/commands.js]: Great point - there is no reason and I will change it to const.

**Comment by stubailo** [comment]
I would also change the name of the app to something simpler - having the words "reactive" and "debugger" might imply some features that it doesn't actually have. I feel like it could just be called "development-error-app" or something

**Comment by anubhav94** [comment]
Good point and fixed.

**Comment by Slava** [comment]
The new rebased PR is here: https://github.com/meteor/meteor/pull/4881

**Pull request closed without merging**